### PR TITLE
Feature/#56 banks board

### PR DESCRIPTION
**Pull request by paula0403**

### CODE REVIEW

File: src/app/View.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ export class View {
         });
       },
     );
+    console.log(avatarsElements);
```
Comment (Enessetere) [inline src/app/View.ts]: This line should be deleted

**Review by Enessetere — DISMISSED** [review]
Looks good. Wait with merge until #55 will be merged.

**Review by synowa — DISMISSED** [review]
Looks good, although conflicts need to be resolved before merging.
If anybody wants to see how it looks, these lines could be used in app.ts.
```ts
    const bank = new BankBoard();
    const bankView = bank.renderBankBoard(new Bank(mockHerdConfig));
    Render.render('#sf-app', bankView);

File: src/app/components/BankBoard.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { Render } from '../utils/Render';
+import { Bank } from '../logic/Bank';
+import { flatten } from 'lodash';
+export class BankBoard {
+  /* returns Bank board with animals and counts */
+  renderBankBoard(bank: Bank): HTMLElement {
+    const bankView: HTMLElement = Render.elementFactory('div', {
+      className: 'bank',
```
Comment (vieraboschkova) [inline src/app/components/BankBoard.ts]: perhaps bank__board?

File: src/app/components/BankBoard.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+          'div',
+          { className: 'bank__count' },
+          `x${countElement}`,
+        );
+
+        return [animalImg, animalCount];
+      },
+    );
+
+    Render.childrenInjector(
```
Comment (vieraboschkova) [inline src/app/components/BankBoard.ts]: Are these methods repeated in player board? Perhaps we could extract them to a single method and inherit from the other class and use it here?

**Review by vieraboschkova — CHANGES_REQUESTED** [review]
Merge with develop and prepare manual please.

File: src/app/components/BankBoard.ts (hunk 3 above)
Comment (paula0403) [reply src/app/components/BankBoard.ts]: This can be done in refactor if both components stay.

**Review by Enessetere — APPROVED** [review]

**Review by vieraboschkova — APPROVED** [review]

**Pull request merged**